cmake/public/gridtools_setup_targets.cmake

```diff
@@ -177,11 +177,11 @@ macro(_gt_setup_targets _config_mode clang_cuda_mode)
     endif()
 
     if (GT_CUDA_TYPE STREQUAL NVCC-CUDA)
-        set(GT_CUDA_ARCH_FLAG -arch)
+        set(GT_CUDA_ARCH_FLAG "-arch" CACHE INTERNAL "Compute capability flag name")
     elseif (GT_CUDA_TYPE STREQUAL Clang-CUDA)
-        set(GT_CUDA_ARCH_FLAG --cuda-gpu-arch)
+        set(GT_CUDA_ARCH_FLAG "--cuda-gpu-arch" CACHE INTERNAL "Compute capability flag name")
     elseif (GT_CUDA_TYPE STREQUAL HIPCC-AMDGPU)
-        set(GT_CUDA_ARCH_FLAG --amdgpu-target)
+        set(GT_CUDA_ARCH_FLAG "--amdgpu-target" CACHE INTERNAL "Compute capability flag name")
```

Comment:
Compute capability is an NVIDIA-only term. But as this is INTERNAL anyway, probably nobody cares…